<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UiCamera</class>
 <widget class="QWidget" name="UiCamera">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>268</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>251</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="vLayout_main">
    <item>
     <widget class="QPushButton" name="btn_Camera">
      <property name="maximumSize">
       <size>
        <width>230</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>&gt;基础摄像机设置</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_camera" native="true">
      <widget class="QLabel" name="label_25">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>230</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>裁剪起始/m:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>类型:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>视野（度）:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="combo_cameraType">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>10</y>
         <width>60</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>透视</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>正交</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>全景</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_26">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>280</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>结束/m:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_23">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>120</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>水平移位:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_24">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>170</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>垂直移位:</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>30</y>
         <width>121</width>
         <height>301</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="vLayout_camera">
        <item>
         <widget class="QLineEdit" name="lineEdit_cameraFOV">
          <property name="text">
           <string>45</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_cameraYaw">
          <property name="text">
           <string>-90</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_cameraPitch">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_clipNear">
          <property name="text">
           <string>0.1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_clipFar">
          <property name="text">
           <string>100</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
